README.md

```diff
@@ -1 +1,203 @@
-docs/01-readme.md
```

Comment:
The symlink is not useless. By linking, rather than copying the file, we don't have to make identical changes in multiple files.

In fact, your second commit here is a good example for that.

Reply:
I mean, then why do we have a copy of the readme in the docs folder anyways? Doesn't seem to contribute much.

Reply:
The file at `docs/01-readme.md` exists, so that [this page](https://awesomewm.org/apidoc/documentation/01-readme.md.html) exists in the official docs.
The link at `README.md` exists to satisfy GitHub's convention where the repo's introductory reading should be.

And since we don't want to have two copies that we would need to update individually, we link one to the other.